src/item.ts

```diff
@@ -4,5 +4,14 @@ export interface JSONItem {
     name: string;
     data: { [k: string]: string };
     quantity: number;
+    available_quantity: number;
```

Comment:
I'm actually thinking that we should have a `total_quantity` and a `assigned_quantity` fields instead of `available_quantity`. I think it's better to track how many have been assigned instead of how many are left to assign